samples/Sentry.Samples.Ios/Sentry.Samples.Ios.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net9.0-ios18.0</TargetFramework>
+    <TargetFramework>net9.0-ios</TargetFramework>
```

Comment:
Although technically we could do this in our Samples, it might break our builds if the Workloads change. Is there any strong reason to remove the `ios18.0` suffix?

Reply:
Apps do not obey the versioning here.  This is only need for libraries.  This also breaks rider detection of the app.